src/onboarding/Deployment/DeploymentStepsPanel.js

```diff
@@ -65,10 +68,19 @@ function DeploymentStepsPanel({ transactionsStatus, pending, allSuccess }) {
       </div>
 
       {!allSuccess && (
-        <Info>
+        <Info mode="warning">
           It might take some time before these transactions get processed.
-          Please be patient and{' '}
-          <strong>do not close this window until it finishes.</strong>
+          Please be patient and do not close this window until it finishes.
+          {provider === 'metamask'
+            ? ' Additionally, do not use the "Speed Up" MetaMask feature, otherwise it will result in configuration errors. '
```

Comment:
Perhaps we can link to the particular feature we are addressing, if metamask has documentation on it?

As well, "result in configuration errors" may be a bit misleading; we may want to say "you may not be able to finish creating your organization".